checks/evaluation/webhooks_test.go

```diff
@@ -17,136 +17,210 @@ package evaluation
-							ID:             1234,
-							UsesAuthSecret: true,
-						},
+			findings: []finding.Finding{
+				{
+					Probe:   "webhooksWithoutSecret",
+					Outcome: finding.OutcomePositive,
+					Values: map[string]int{
+						"totalWebhooks": 1,
 					},
 				},
 			},
-			want: checker.CheckResult{
+			result: scut.TestReturn{
 				Score: 10,
 			},
 		},
 		{
-			name: "1 webhook with no secret",
-			args: args{
-				name: "1 webhook with no secret",
-				dl:   &scut.TestDetailLogger{},
-				r: &checker.WebhooksData{
-					Webhooks: []clients.Webhook{
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/1234",
-							ID:             1234,
-							UsesAuthSecret: false,
-						},
+			name: "2 webhooks one of which has secret",
+			findings: []finding.Finding{
+				{
+					Probe:   "webhooksWithoutSecret",
+					Outcome: finding.OutcomeNegative,
+					Values: map[string]int{
+						"totalWebhooks": 2,
 					},
 				},
 			},
-			want: checker.CheckResult{
-				Score: 0,
+			result: scut.TestReturn{
+				Score: 5,
 			},
 		},
 		{
-			name: "many webhooks with no secret and with secret",
-			args: args{
-				name: "many webhooks with no secret and with secret",
-				dl:   &scut.TestDetailLogger{},
-				r: &checker.WebhooksData{
-					Webhooks: []clients.Webhook{
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/1234",
-							ID:             1234,
-							UsesAuthSecret: false,
-						},
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/1111",
-							ID:             1111,
-							UsesAuthSecret: true,
-						},
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/4444",
-							ID:             4444,
-							UsesAuthSecret: true,
-						},
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/3333",
-							ID:             3333,
-							UsesAuthSecret: false,
-						},
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/2222",
-							ID:             2222,
-							UsesAuthSecret: false,
-						},
+			name: "Five webhooks three of which have secrets",
+			findings: []finding.Finding{
+				{
+					Probe:   "webhooksWithoutSecret",
+					Outcome: finding.OutcomeNegative,
+					Values: map[string]int{
+						"totalWebhooks": 5,
+					},
+				},
+				{
+					Probe:   "webhooksWithoutSecret",
+					Outcome: finding.OutcomeNegative,
+					Values: map[string]int{
+						"totalWebhooks": 5,
 					},
 				},
 			},
-			want: checker.CheckResult{
+			result: scut.TestReturn{
 				Score: 6,
 			},
 		},
+		{
+			name: "One of 12 webhooks does not have secrets",
+			findings: []finding.Finding{
+				{
+					Probe:   "webhooksWithoutSecret",
+					Outcome: finding.OutcomeNegative,
+					Values: map[string]int{
+						"totalWebhooks": 12,
+					},
+				},
+			},
+			result: scut.TestReturn{
+				Score: 9,
+			},
+		},
+		{
+			name: "12/12 webhooks do not have secrets",
```

Comment:
nit: the test name should reflect the purpose of the test, not describe what it's doing. So based on laurent's comment that led to the test it would be something like:

"score doesn't drop below min score"